### DISCUSSION TITLE
Add `object-fit: cover;` to the avatar CSS in the CP to prevent incorrect resizing

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4175] [Feature Requests]
### Suggest an idea for ExpressionEngine

Avatar img is currently constrained to a square. Could be great to add here `object-fit: cover;` to ensure it correctly displays images of all aspect ratios

![Screenshot 2024-03-26 110715](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/f236ba3f-4244-4c51-9684-bf5173d88468)

**Comment by intoeetive** [comment]
@Yulyaswan fyi

**Comment by Yulyaswan** [comment]
PR #4181

**Comment by bryannielsen** [comment]
This was added in [7.4.6](https://docs.expressionengine.com/latest/installation/changelog.html#version-746)
